llvm/test/Transforms/LoopVectorize/RISCV/reg-usage-bf16.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -passes=loop-vectorize -mtriple riscv64 -mattr=+v,+zvfbfmin -debug-only=loop-vectorize -riscv-v-register-bit-width-lmul=1 -S < %s 2>&1 | FileCheck %s
```

Comment:
Need `REQUIRES: asserts`

Reply:
Done in 30d7dcc1db476b828d3b0a7b5c9bcfbe5bd5f216